app/views/child_benefit_tax/_money_input.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render "govuk_publishing_components/components/input", {
```

Comment:
Whilst income isn't a known number of characters, I wonder if it's worth making the inputs narrower so they don't take up the full width of the screen. 

This is what a `width` of `10` looks like:

<img width="681" alt="Screenshot 2019-11-12 at 08 58 57" src="https://user-images.githubusercontent.com/1732331/68656990-ba43eb00-052a-11ea-816e-5028fc67195f.png">

Reply:
Great shout, I'd overlooked that option, thanks.